### DISCUSSION TITLE
How do the "overlaid half-open views" work?

### DISCUSSION THREAD

**Discussion by Wittmaxi** [eclipse-platform/eclipse.platform.ui#2236] [General]
How does this "half folded out" view work? 
![grafik](https://github.com/user-attachments/assets/78f50d8c-1854-4eea-80b8-354014d1f912)
(here: the search view)
How is it rendered and would this be possibly a cleaner way to implement the Find/Replace Overlay instead of creating a new shell shadowing the editor?

**Comment by wimjongman** [comment]
What you see is the fly-out view.

Select the viewstack in the trim area an tick "Show in the original location"


![image](https://github.com/user-attachments/assets/2da04572-10e7-4774-a935-6c44ccc55529)
